components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -108,35 +111,75 @@ export default {
 		isQuiet: false,
 		isDisabled: false,
 		isFocused: false,
-		isKeyboardFocused: false,
 		iconType: "ui",
 		iconName: "ChevronDown",
-		position: "right"
+		position: "right",
 	},
 };
 
 export const Default = Template.bind({});
 Default.args = {};
 
 export const WithLabel = Template.bind({});
+WithLabel.storyName = "With label";
 WithLabel.args = {
 	label: "Select",
 };
 
 export const Disabled = Template.bind({});
-Disabled.tags = ["vrt-only"];
+Disabled.tags = ["!dev"];
```

Comment:
Did we want to include the same sidebar items as on prod? Storybook prod has default, disabled, express, quiet (with label), and with label all listed.

Reply:
I don't think all of these need to be there anymore (especially Express). We've been moving away from having stories for every single state and combination of states, and condensing as much as possible into the Default. And handling the large number of possible states in VRTs and Chromatic-only templates. Unless we're changing our approach there. I think the team still needs to have that conversation about conventions for this reorganization.